### PR TITLE
MES-1150 - Welcome Page Updates

### PR DESCRIPTION
**Pull request by Cavalier07**
## Description and relevant Jira numbers

This PR adds the new design for the welcome page in ticket MES-1150

unadjustednonraw_thumb_2c

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [x] PO's approval

### CODE REVIEW

**Comment by ammarhaiderbjss** [comment]
can't see the new buttons in the component library? https://wiki.i-env.net/display/MES/Component+library

File: src/pages/welcome-page/welcome-page.scss [hunk 1]
```diff
@@ -1,8 +1,71 @@
...
-  .welcome-message {
-    font-family: GDSTransportWebsite-Bold;
-    text-align: center;
-    padding-top: 3rem;
-    font-size: 3rem;
+  font-weight: bold;
+
+  ion-content {
+    background-image: url('/assets/imgs/welcome-page-background.png');
+    -webkit-background-size: cover;
```
Comment (johnmorleybjss) [inline src/pages/welcome-page/welcome-page.scss]: The various vendor prefixes here are probably a good candidate for a Sass mixin.

**Comment by ammarhaiderbjss** [comment]
image
1.3 MB for the background image?!

**Comment by Cavalier07** [comment]
> image
> 1.13 MB for the background image?!

Have dropped it down to 100kb - do we still need it smaller?

**Comment by ammarhaiderbjss** [comment]
Fine for now 👍

File: src/pages/welcome-page/welcome-page.scss [hunk 2]
```diff
@@ -1,8 +1,71 @@
...
+    letter-spacing: 0.46px;
+  }
+  .title {
+    color: map-get($colors, white);
+    font-size: 48px;
+    line-height: 50px;
+  }
+  .text {
+    color: map-get($colors, white);
+    margin: -20px 0 0 -40px;
```
Comment (ammarhaiderbjss) [inline src/pages/welcome-page/welcome-page.scss]: What's the minus margin for?

File: src/pages/welcome-page/welcome-page.scss [hunk 3]
```diff
@@ -1,8 +1,71 @@
...
+    height: 138px;
+    border-radius: 10px;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
+    background-image: linear-gradient(to bottom, map-get($colors, white), #efefef);
+    &:active {
+      background-image: linear-gradient(to bottom, #fdfdfd, #cbcbcb);
+    }
+    span {
+      @extend .text;
+      margin: 0 0 0 -125px;
```
Comment (ammarhaiderbjss) [inline src/pages/welcome-page/welcome-page.scss]: -margin again here

**Comment by Cavalier07** [comment]
> can't see the new buttons in the component library? https://wiki.i-env.net/display/MES/Component+library

It's there :)

**Pull request merged**